scripts/flytekit_spark_entrypoint.sh

```diff
@@ -24,7 +24,7 @@ myuid=$(id -u)
 mygid=$(id -g)
 # turn off -e for getent because it will return error code in anonymous uid case
 set +e
-uidentry=$(getent passwd "$myuid")
+uidentry=$(getent passwd $myuid)
```

Comment:
comments throughout

Reply:
I am tagging this as deprecated in favor of official spark entrypoint in the spark distribution. We can also remove this if no-one is using it (given its broken in its current form).